task/thermal/src/control.rs

```diff
@@ -56,16 +57,25 @@ pub enum FanControl {
 }
 
 impl FanControl {
-    fn set_pwm(&self, fan: Fan, pwm: PWMDuty) -> Result<(), ResponseCode> {
+    fn set_pwm(
+        &self,
+        fan: drv_i2c_devices::max31790::Fan,
```

Comment:
Did you intend to make this a fully qualified type, vs just `Fan`?

In `main.rs`, we're already re-exporting that type
```rust
pub use drv_i2c_devices::max31790::Fan;
```

and importing it at the top of this module
```rust
use crate::{bsp::BspT, Fan, ThermalError, Trace};
```

Reply:
Never mind, I see that the crate `Fan` type has changed!